scripts/eurostat/life_expectancy_by_age_gender/demo_r_mlifexp_country.tmcf

```diff
@@ -0,0 +1,3096 @@
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_13Years_Male
+
+Node: E:lifexp->E24
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_14Years_geo
+
+Node: E:lifexp->E25
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_14Years
+observationAbout: E:lifexp->E24
+observationDate: C:lifexp->LifeExpectancy_Person_14Years_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_14Years
+
+Node: E:lifexp->E26
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_14Years_Female_geo
+
+Node: E:lifexp->E27
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_14Years_Female
+observationAbout: E:lifexp->E26
+observationDate: C:lifexp->LifeExpectancy_Person_14Years_Female_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_14Years_Female
+
+Node: E:lifexp->E28
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_14Years_Male_geo
+
+Node: E:lifexp->E29
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_14Years_Male
+observationAbout: E:lifexp->E28
+observationDate: C:lifexp->LifeExpectancy_Person_14Years_Male_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_14Years_Male
+
+Node: E:lifexp->E30
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_15Years_geo
+
+Node: E:lifexp->E31
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_15Years
+observationAbout: E:lifexp->E30
+observationDate: C:lifexp->LifeExpectancy_Person_15Years_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_15Years
+
+Node: E:lifexp->E32
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_15Years_Female_geo
+
+Node: E:lifexp->E33
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_15Years_Female
+observationAbout: E:lifexp->E32
+observationDate: C:lifexp->LifeExpectancy_Person_15Years_Female_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_15Years_Female
+
+Node: E:lifexp->E34
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_15Years_Male_geo
+
+Node: E:lifexp->E35
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_15Years_Male
+observationAbout: E:lifexp->E34
+observationDate: C:lifexp->LifeExpectancy_Person_15Years_Male_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_15Years_Male
+
+Node: E:lifexp->E36
+typeOf: dcs:Country
+dcid: C:lifexp->LifeExpectancy_Person_16Years_geo
+
+Node: E:lifexp->E37
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:LifeExpectancy_Person_16Years
+observationAbout: E:lifexp->E36
+observationDate: C:lifexp->LifeExpectancy_Person_16Years_year
+measurementMethod: EurostatRegionalStatistics
+value: C:lifexp->LifeExpectancy_Person_16Years
+
+Node: E:lifexp->E38
+typeOf: dcs:Country
```

Comment:
Super nit. Country is a schema.org entity, so we prefer schema:Country.

Reply:
removed the geo nodes.